src/Components/Web/src/PublicAPI.Unshipped.txt

```diff
@@ -34,7 +39,31 @@ Microsoft.AspNetCore.Components.Web.HtmlRendering.HtmlRootComponent.HtmlRootComp
 Microsoft.AspNetCore.Components.Web.HtmlRendering.HtmlRootComponent.QuiescenceTask.get -> System.Threading.Tasks.Task!
 Microsoft.AspNetCore.Components.Web.HtmlRendering.HtmlRootComponent.ToHtmlString() -> string!
 Microsoft.AspNetCore.Components.Web.HtmlRendering.HtmlRootComponent.WriteHtmlTo(System.IO.TextWriter! output) -> void
+Microsoft.AspNetCore.Components.Web.RenderMode
+Microsoft.AspNetCore.Components.Web.RenderModeAutoAttribute
+Microsoft.AspNetCore.Components.Web.RenderModeAutoAttribute.RenderModeAutoAttribute() -> void
+Microsoft.AspNetCore.Components.Web.RenderModeAutoAttribute.RenderModeAutoAttribute(bool prerender) -> void
+Microsoft.AspNetCore.Components.Web.RenderModeServerAttribute
+Microsoft.AspNetCore.Components.Web.RenderModeServerAttribute.RenderModeServerAttribute() -> void
+Microsoft.AspNetCore.Components.Web.RenderModeServerAttribute.RenderModeServerAttribute(bool prerender) -> void
+Microsoft.AspNetCore.Components.Web.RenderModeWebAssemblyAttribute
+Microsoft.AspNetCore.Components.Web.RenderModeWebAssemblyAttribute.RenderModeWebAssemblyAttribute() -> void
+Microsoft.AspNetCore.Components.Web.RenderModeWebAssemblyAttribute.RenderModeWebAssemblyAttribute(bool prerender) -> void
+Microsoft.AspNetCore.Components.Web.ServerRenderMode
+Microsoft.AspNetCore.Components.Web.ServerRenderMode.Prerender.get -> bool
+Microsoft.AspNetCore.Components.Web.ServerRenderMode.ServerRenderMode() -> void
+Microsoft.AspNetCore.Components.Web.ServerRenderMode.ServerRenderMode(bool prerender) -> void
+Microsoft.AspNetCore.Components.Web.WebAssemblyRenderMode
+Microsoft.AspNetCore.Components.Web.WebAssemblyRenderMode.Prerender.get -> bool
+Microsoft.AspNetCore.Components.Web.WebAssemblyRenderMode.WebAssemblyRenderMode() -> void
+Microsoft.AspNetCore.Components.Web.WebAssemblyRenderMode.WebAssemblyRenderMode(bool prerender) -> void
```

Comment:
All of these (and some of the ones below) can be removed when the Razor compiler is updated to support the `@rendermode` directive.